src/t_stream.c

```diff
@@ -1416,11 +1416,6 @@ void xreadCommand(client *c) {
         int moreargs = c->argc-i-1;
         char *o = c->argv[i]->ptr;
         if (!strcasecmp(o,"BLOCK") && moreargs) {
-            if (c->flags & CLIENT_LUA) {
-                /* There is no sense to use BLOCK option within LUA */
-                addReplyErrorFormat(c, "%s command is not allowed with BLOCK option from scripts", (char *)c->argv[0]->ptr);
-                return;
-            }
```

Comment:
i guess for backwards compatibility, we can keep this check (and the no-scripts flag for the other blocking commands). so that they'll still respond with an error rather than an empty response.

if we have test coverage for the other DENY_BLOCKING modes (MULTI), we should be safe.

Reply:
@oranagra it looked to me like it's not an issue and this unified the behavior. But I can change it back if you disagree.